src/CompositeException.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\ErrorHandler;
+
+use Exception;
+use Throwable;
+
+/**
+ * Aggregate multiple exceptions into one.
+ */
+final class CompositeException extends Exception
```

Comment:
Maybe instead composite exception create `ApplicationErrorDispatchException` with usage of standard previous exception argument?

And in `HtmlRenderrer` add support of any previous exceptions (`Throwable::getPrevious()`)?

Reply:
I don't get what you want. Can you create PR with clear explanation?

`usage of standard previous exception argument`
Multiple exceptions are not supported by PHP as a previous stage of an exception.